src/sentry/services/hybrid_cloud/organization/model.py

```diff
@@ -154,12 +154,15 @@ def get_audit_log_metadata(self, user_email: Optional[str] = None) -> Mapping[st
         }
 
 
-class RpcOrganizationFlags(RpcModel):
+class RpcOrganizationMappingFlags(RpcModel):
+    early_adopter: bool = False
+    require_2fa: bool = False
```

Comment:
This seems a little odd semantically. Is deduplicating this totally necessary? Alternatively, is there any reason we wouldn't want to just replicate all of the flags by default?

Reply:
i vote to replicate all flags

Reply:
Hmm.  Ok.  It isn't much cost to replicate all the flags I suppose.

Reply:
I, am going to replicate more flags, but I decided to keep the difference between `OrganizationMappingFlags` and `OrganizationFlags`.  The reality is that if one introduces a new organization flag, there is no atomic way for that to immediately work for organization mapping -- it will always require time between introducing the new flag, and then introducing the replication to provide it.  Keeping these types separate ensures that only when we guarantee replication of a flag is it available from OrganizationMapping side.  An explicit action is always required to implement this and it is intentional.

Reply:
That seems like a fair compromise, sorry for the extra work 😅

Reply:
I don't know if we'll end up using flags like `disable_new_visibility_features` in control  (as we don't even use in the application today). The same is true for `codecov_access`.

Reply:
I do tend to agree.  I decided to start by replicating all flags for now so that we don't get any open questions about them.  There is minimal cost to having it stored even if it's not useful, but it does increase interpretability.